### PR TITLE
Remove `DecoratedEnumerableProxy`

### LINKED ISSUE #35: Clean code related for ActiveAdmin
Since the latest release of ActiveAdmin allow to use PORO decorator, we may clean some part of `dekorator` code.

https://github.com/activeadmin/activeadmin/pull/6249

### PR DESCRIPTION
**Pull request by nicolas-brousse**
Closes  #35

### CODE REVIEW

**Comment by nicolas-brousse** [comment]
@Spone Should we update the README section about ActiveAdmin? Or just remove it?

**Comment by Spone** [comment]
> @Spone Should we update the README section about ActiveAdmin? Or just remove it?

An update is probably better, I guess?

File: README.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ decorated_user.posts.first # => ArticleDecorator
 
 ### ActiveAdmin
 
-This gem is compatible with [`activeadmin`][activeadmin].
+This gem is compatible with [`activeadmin`][activeadmin] ([2.8+](https://github.com/activeadmin/activeadmin/pull/6249)).
```
Comment (Spone) [inline README.md:159]: Maybe add a mention of the gem version compatible with ActiveAdmin < 2.8?

Comment (nicolas-brousse) [reply README.md]: Updated

**Review by Spone — APPROVED** [review]
Always great to remove code :)

**Pull request merged**